json-webhook/README.md

```diff
@@ -0,0 +1,62 @@
+# JSON Webhook Adapter
+
+Convert a Twilio webhook event into a JSON webhook, suitable for services like Zapier and IFTTT.
```

Comment:
This might be just a semantic issue, but this description is a little confusing to me. We aren't exactly converting anything on twilio's side, we are using Twilio SMS webhook to forward the message to a third party webhook like Zapier or IFTTT